Parsers/CiscoMeraki/CiscoMeraki.txt

```diff
@@ -0,0 +1,156 @@
+        SrcPortNumber = toint(extract(@"(peer_contact|ip_src)=\'([0-9\.]+)\:(\d+)\'",3,Substring)),
+        PeerIdentity = tostring(extract(@"peer_ident=\'(\S+)\'",1,Substring)),
+        Radio = toint(extract(@"radio=\'(\d+)\'",1,Substring)),
+        Vap = toint(extract(@"vap=\'(\S+)\'\s",1,Substring)),
+        Group = toint(extract(@"group=\'(\S+)?\'",1,Substring)),
+        Attribute = toint(extract(@"attr=\'(\S+)?\'",1,Substring)),
+        ClientMacAddr = tostring(extract(@"client_mac=\'(\S+)\'",1,Substring)),
+        Channel = toint(extract(@"channel=\'(\S+)\'",1,Substring)),
+        Reason = toint(extract(@"reason=\'(\S+)\'",1,Substring)),
+        AppleDaReason = toint(extract(@"apple_da_reason=\'(\S+)\'",1,Substring)),
+        Instigator = toint(extract(@"instigator=\'(\S+)\'",1,Substring)),
+        Duration = tostring(extract(@"duration=\'(\S+)\'",1,Substring)),
+        FullConn = tostring(extract(@"full_conn=\'(\S+)\'",1,Substring)),
+        IpResp = tostring(extract(@"ip_resp=\'(\S+)\'\s",1,Substring)),
+        HttpResp = tostring(extract(@"http_resp=\'(\S+)\'",1,Substring)),
+        ArpResp = tostring(extract(@"arp_resp=\'(\S+)\'",1,Substring)),
+        ArpSrcIpAddr = tostring(extract(@"arp_src=\'(\S+)\'",1,Substring)),
+        Connectivity = tostring(extract(@"connectivity=\'(\S+)\'",1,Substring)),
+        Rtt = tostring(extract(@"rtt=\'([\w+\.\s]+)\'",1,Substring)),
+        UserName = tostring(extract(@"identity=\'(\S+)\'",1,Substring)),
+        Aid = tostring(extract(@"aid=\'(\S+)\'",1,Substring)),
+        Spi = tostring(extract(@"spi=(\S+)$",1,Substring)),
+        DvcMacAddr = tostring(extract(@"device=\'(\S+)\'",1,Substring)),
+        State = tostring(extract(@"state=\'(\S+)\'",1,Substring)),
+        AlarmId = toint(extract(@"alarm_id=\'(\S+)\'",1,Substring)),
+        DosCount = tostring(extract(@"dos_count=\'(\S+)\'",1,Substring)),
+        InterArrival = tostring(extract(@"inter_arrival=\'(\S+)\'",1,Substring))
+| extend IpAddr = tostring(extract(@"dhcp lease of ip ([\d\.]+)", 1, Substring)),
+        ServerMacAddr = tostring(extract(@"server mac ([\w\:]+)", 1, Substring)),
+        ClientMacAddr = iif(isempty(ClientMacAddr), tostring(extract(@"client mac ([\w\:]+)", 1, Substring)), ClientMacAddr),
+        RouterIpAddr = tostring(extract(@"router ([\d\.]+)", 1, Substring)),
+        Subnet = tostring(extract(@"subnet ([\d\.]+)", 1, Substring)),
+        Dns = split(extract(@"dns ([\d\.\,\:\s]+)", 1, Substring), ", "),
+        SrcPortNumber = iif(isempty(SrcPortNumber), toint(extract(@"port=\'(\S+)\'",1,Substring)), SrcPortNumber)
+| extend Dns = iif(Dns[0] == "", "", Dns)
+| project-away Substring;
+let AllOtherLogs = LogHeader
+| where LogType !in ("urls", "airmarshal_events","security_event","ids-alerts", "events") and LogType !contains "flow";
+union UrlEvents, FlowEvents, AirMarshallEvents, SecurityEvents, IDSAlerts, EventLogs, AllOtherLogs
```

Comment:
Whatever you change the name to above, remember to change SecurityEvents here also.

Reply:
updates made